packages/editor/src/plugins/image/components/licence-dropdown.tsx

```diff
@@ -0,0 +1,111 @@
+import { EditorTooltip } from '@editor/editor-ui/editor-tooltip'
+import { faQuestionCircle } from '@fortawesome/free-regular-svg-icons'
+import { faCaretDown, faCaretUp } from '@fortawesome/free-solid-svg-icons'
+import * as Select from '@radix-ui/react-select'
+import React, { useState, useEffect, useRef } from 'react'
+
+import { FaIcon } from '@/components/fa-icon'
+import { useEditorStrings } from '@/contexts/logged-in-data-context'
+import { cn } from '@/helper/cn'
+
+interface LicenseDropdownProps {
+  defaultLicense?: string
+  onLicenseChange?: (license: string) => void
+  isPixabayImage?: boolean
+}
+
+export const LicenseDropdown: React.FC<LicenseDropdownProps> = ({
```

Comment:
only usage of React.FC in this codebase.

Reply:
Removed